packages/sample-app/README.md

```diff
@@ -7,19 +7,19 @@ A very basic React app featuring Babel and HMR. Performs all the nifty optimizat
 
 ```sh
 git clone https://github.com/andywer/webpack-blocks.git
-cd webpack-blocks/test-app
-npm install
+yarn
```

Comment:
See [this comment](https://github.com/andywer/webpack-blocks/pull/273/files#diff-6a3371457528722a734f3c51d9238c13R8)